src/app/wallet/wallet.component.ts

```diff
@@ -80,6 +84,7 @@ export class WalletComponent implements OnInit {
     refreshTransactionHistory() {
         this.transactionHistoryPage = this.transactionHistory
             .map((transaction, i) => ({id: i + 1, ...transaction}))
-            .slice((this.tablePage - 1) * this.tablePageSize, (this.tablePage - 1) * this.tablePageSize + this.tablePageSize);
+            .slice((this.tablePage - 1) * this.tablePageSize, (this.tablePage - 1) * this.tablePageSize + this.tablePageSize)
+            
```

Comment:
Keep using semicolons everywhere. The linter will cry.
```suggestion
            .slice((this.tablePage - 1) * this.tablePageSize, (this.tablePage - 1) * this.tablePageSize + this.tablePageSize);
```

Reply:
yes, I noticed this @mdrusko1 - you don't need semicolons in TypeScript. aamof they're discouraged

Reply:
Mea culpa, keep those semis active :D